Custom Unit Description/CustomUnitDescription.js

```diff
@@ -27,6 +27,27 @@ tau.mashups
                 ],
                 model: 'description:Description',
                 sampleData: { description: '<div>Entity description</div>' }
+           },
+           {
+                id: 'custom_rich_text_list',
+                classId: 'tau-board-unit_type_custom-rich-text',
+                name: 'Description',
+                types: [ types.PROJECT, types.FEATURE, types.EPIC, types.STORY, types.TASK, types.BUG, types.REQUEST ],
+                sections: 1,
+                sizes: [ sizes.LIST ],
+                template: [
+                  '<% if(this.data.description) { %>',
+                    '<div class="tau-board-unit__value">',
+                    '<% var tmp = document.createElement("DIV"); %>',
+                    '<% tmp.innerHTML = this.data.description; %>',
+                    '<% var plain = tmp.textContent || tmp.innerText || ""; %>',
```

Comment:
Inefficient and duplicated code:
- temporary `div` is created then freed for each unit instance
- code is duplicated in two units

Extract duplicated JS code to reusable custom function 

```js
var div;
function extractPlainTextFromRichText(richText) {
    div = div || document.createElement('div');
    div.innerHTML = richText;
    return div.textContent || div.innerText || '';
}

function getPlainDescription(description, maxLength) {
    return _.truncate(extractPlainTextFromRichText(description), maxLength).replace(/\u00a0/g, ' ');
}

...
            template: {
                customFunctions: {
                    getPlainDescription: getPlainDescription
                },
                markup: [
                    ...
                    '<%= fn.getPlainDescription(this.data.description, 150) %>'
                ]
            },

...

            template: {
                customFunctions: {
                    getPlainDescription: getPlainDescription
                },
                markup: [
                    ...
                    '<%= fn.getPlainDescription(this.data.description, 300) %>'
                ]
            },

```